website/docs/r/appautoscaling_scheduled_action.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+layout: "aws"
+page_title: "AWS: aws_appautoscaling_scheduled_action"
+sidebar_current: "docs-aws-resource-appautoscaling-scheduled-action"
+description: |-
+  Provides an Application AutoScaling ScheduledAction resource.
+---
+
+# aws_appautoscaling_scheduled_action
+
+Provides an Application AutoScaling ScheduledAction resource.
+
+## Example Usage
+
+### DynamoDB Table Autoscaling
+
+```hcl
+resource "aws_appautoscaling_target" "dynamodb" {
+  max_capacity       = 100
+  min_capacity       = 5
+  resource_id        = "table/tableName"
+  role_arn           = "${data.aws_iam_role.DynamoDBAutoscaleRole.arn}"
+  scalable_dimension = "dynamodb:table:ReadCapacityUnits"
+  service_namespace  = "dynamodb"
+}
+
+resource "aws_appautoscaling_scheduled_action" "dynamodb" {
+  name = "dynamodb"
+  service_namespace = "${aws_appautoscaling_target.dynamodb.service_namespace}"
+  resource_id = "${aws_appautoscaling_target.dynamodb.resource_id}"
+  scalable_dimension = "${aws_appautoscaling_target.dynamodb.scalable_dimension}"
+  schedule = "at(2006-01-02T15:04:05)"
+
+  scalable_target_action {
+    min_capacity = 1
+    max_capacity = 200
+  }
+}
+```
+
+### ECS Service Autoscaling
+
+```hcl
+resource "aws_appautoscaling_target" "ecs" {
+  max_capacity       = 4
+  min_capacity       = 1
+  resource_id        = "service/clusterName/serviceName"
+  role_arn           = "${var.ecs_iam_role}"
+  scalable_dimension = "ecs:service:DesiredCount"
+  service_namespace  = "ecs"
+}
+
+resource "aws_appautoscaling_scheduled_action" "ecs" {
+  name = "ecs"
+  service_namespace = "${aws_appautoscaling_target.ecs.service_namespace}"
+  resource_id = "${aws_appautoscaling_target.ecs.resource_id}"
+  scalable_dimension = "${aws_appautoscaling_target.ecs.scalable_dimension}"
+  schedule = "at(2006-01-02T15:04:05)"
+
+  scalable_target_action {
+    min_capacity = 1
+    max_capacity = 10
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the scheduled action.
+* `service_namespace` - (Required) The namespace of the AWS service. Documentation can be found in the parameter at: [AWS Application Auto Scaling API Reference](https://docs.aws.amazon.com/ApplicationAutoScaling/latest/APIReference/API_PutScheduledAction.html#ApplicationAutoScaling-PutScheduledAction-request-ServiceNamespace) Valid Values: ecs | elasticmapreduce | ec2 | appstream | dynamodb
```

Comment:
I would avoid documenting all valid values as that will become a maintenance burden as these expand. How about we just mention a few examples? The user can always IMO look up all valid values in the docs, right?